src/back/auth/src/repo/table_session.cpp

```diff
@@ -26,15 +28,9 @@ const storages::postgres::Query kQuerySave{
 
 void Session::Save(const model::Session& data)
 {
-	storages::postgres::Transaction t =
-		_pg->Begin("insert_session_transaction",
-			ClusterHostType::kMaster, {});
-
-	std::apply([&t](const auto&... args) {
-		t.Execute(kQuerySave, args...);
+	std::apply([this](const auto&... args) {
+		this->_db->Execute(kQuerySave, args...);
```

Comment:
Тут this кажется лишний.
```suggestion
		_db->Execute(kQuerySave, args...);
```

Reply:
Без передачи this в эту функцию (анонимную?) не получилось завести код, передавать чисто ссылку на БД тоже не получилось. Ошибка "на вложенную функцию "this" нельзя ссылаться внутри тела лямбды, если она не находится в списке записей". См. [https://ru.stackoverflow.com/questions/1466532/%D0%BD%D0%B0-%D0%B2%D0%BB%D0%BE%D0%B6%D0%B5%D0%BD%D0%BD%D1%83%D1%8E-%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D1%8E-this-%D0%BD%D0%B5%D0%BB%D1%8C%D0%B7%D1%8F-%D1%81%D1%81%D1%8B%D0%BB%D0%B0%D1%82%D1%8C%D1%81%D1%8F-%D0%B2%D0%BD%D1%83%D1%82%D1%80%D0%B8-%D1%82%D0%B5%D0%BB%D0%B0-%D0%BB%D1%8F%D0%BC%D0%B1%D0%B4%D1%8B-%D0%B5%D1%81%D0%BB%D0%B8-%D0%BE%D0%BD%D0%BE-%D0%BD%D0%B5-%D0%BD%D0%B0%D1%85](url)